### PR TITLE
Reworking how aborts work and adding "restore cryo" button

### PR DESCRIPTION
**Pull request by lisazacarias**

### CODE REVIEW

File: q0.ui [hunk 1]
```diff
@@ -266,22 +269,36 @@
...
-            <double>10.000000000000000</double>
+          </widget>
+         </item>
+         <item row="1" column="1">
+          <widget class="PyDMSpinbox" name="jt_setpoint_spinbox">
+           <property name="toolTip">
+            <string/>
+           </property>
+           <property name="maximum">
+            <double>75.000000000000000</double>
```
Comment (nneveu) [inline q0.ui:301]: Should this also be 160? or stay 75 here?

File: q0_linac.py [hunk 2]
```diff
@@ -709,14 +722,16 @@ def takeNewCalibration(self, jt_search_start: datetime = None,
         
         self.heater_power = self.valveParams.refHeatLoadDes
         
-        print("Restoring initial cryo conditions")
```
Comment (nneveu) [inline q0_linac.py:712]: I don't think it hurts to have the info printed here? if you are still doing a 'restore' in the next line

**Review by nneveu — APPROVED** [review]
Yay, code reviews! Looks nice!

File: q0.ui (hunk 1 above)
Comment (lisazacarias) [reply q0.ui:301]: Different system! The JT doesn't actually go below 5% or above 70%

File: q0_linac.py (hunk 2 above)
Comment (lisazacarias) [reply q0_linac.py:712]: Info like, what it's doing?

File: q0.ui (hunk 1 above)
Comment (nneveu) [reply q0.ui:301]: so should the max be 70%?

File: q0_linac.py (hunk 2 above)
Comment (nneveu) [reply q0_linac.py:712]: yea looks like you're removing the print statement? I think it can stay in?

**Review by nneveu — APPROVED** [review]

File: q0_linac.py (hunk 2 above)
Comment (lisazacarias) [reply q0_linac.py:712]: Oh because that gets printed in the lcld-tools function!

**Pull request merged**